packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/__snapshots__/react-ssr-wrapper.spec.tsx.snap

```diff
@@ -1158,7 +1158,6 @@ exports[`manual test cases should pass internal props correctly to p-tabs-item 1
               >
                 <div
                   class="scroll-wrapper"
-                  role="tablist"
```

Comment:
Shouldn't this attribute stay since the children are `button` elements? I believe also the other scenario with `a` elements is missing?

Reply:
Manual test for `Tabs Bar` with slotted `a` tags created. 
`role="tablist"` won't be fixed because it has no visual effect.